.rubocop_todo.yml

```diff
@@ -1,42 +1,65 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2016-12-14 08:11:19 +0100 using RuboCop version 0.45.0.
+# on 2017-03-12 12:11:26 -0400 using RuboCop version 0.47.1.
```

Comment:
rebase on latest develop to check against latest rubocop version. I would expect no changes to happen in this TODO except rare cases